WBooks/src/app/screens/BookDetail/index.js

```diff
@@ -7,14 +7,16 @@ import styles from './styles';
 
 function BookDetail() {
   return (
-    <ScrollView style={styles.bookDetailContainer}>
-      <View style={styles.detailBoxContainer}>
-        <DetailBox />
-      </View>
-      <View style={styles.commentBoxContainer}>
-        <CommentBox />
-      </View>
-    </ScrollView>
+    <View style={styles.bookDetailContainer}>
```

Comment:
Is really necessary this `View`?

Reply:
Yes, if i remove this View, the commentBox bottom won't see okey. I tried other options but they haven't worked